src/shared/text/generic.ts

```diff
@@ -0,0 +1,10 @@
+/*!
```

Comment:
`src/shared/text/generic.ts` looks like a setup for multiple files containing localized strings. But we could keep all the strings in 1 file.

Reply:
Organizing text by domain is intended to keep it easy to locate strings, and prevent one large mashup of strings (trying to locate text of interest in package.nls.json is a good example).

I do think its easy to start small and have the flexibility to grow it out as needed. What do you suggest?

Reply:
I suggest putting this in `src/shared/localized.ts`. Multiple modules is unnecessary granularity.

> trying to locate text of interest in package.nls.json is a good example

Trying to locate a particular localized string does not become easier by spreading it across many files. It is just the nature of having many localized strings.

Reply:
Will do